src/OrchardCore.Modules/OrchardCore.Queries/Sql/Controllers/AdminController.cs

```diff
@@ -75,7 +75,7 @@ public async Task<IActionResult> Query(AdminQueryViewModel model)
             var stopwatch = new Stopwatch();
             stopwatch.Start();
 
-            var connection = _store.Configuration.ConnectionFactory.CreateConnection();
+            await using var connection = _store.Configuration.ConnectionFactory.CreateConnection();
```

Comment:
Is it awaited?

Reply:
yes.

Reply:
@hishamco At the end of an `using var disposable` it calls `disposable.Dispose()`.

At the end of an `await using var disposable` it calls `await disposable.DisposeAsync()`.

But you can do that only if `disposable` is an `IAsyncDisposable`.